upgrade/breaking-changes/2-0-0.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 2.0.0
+description: Handle the changes in the 2.0.0 release version of the Telerik UI for .NET MAUI components.
+page_title: Breaking Changes in 2.0.0
+slug: changes-in-2-0-0
+position: 1
+---
+
+# Breaking Changes in 2.0.0
+
+This article lists the breaking changes included in Telerik UI for .NET MAUI 2.0.0 release version.
+
+## Common Changes
+
+* *All `Telerik.XamarinForms.*` namespaces are renamed to `Telerik.Maui.Controls.Compatibility.*` .
```

Comment:
* Use a table to present the changes as it will provide better readability. 
* Introduce the table with a sentence
* Use two columns: "Previous namespace" and "Current Namespace".